docs/docs/11_deploy_to_render/06_run_everything_docker_compose/README.md

```diff
@@ -0,0 +1,116 @@
+# How to run the app and database with Docker Compose
+
+Up until now we've been running `docker-compose up` to start the REST API container.
```

Comment:
I think the docker compose command does not have a "`-`" anymore.

```suggestion
Up until now we've been running `docker compose up` to start the REST API container.
```

Reply:
You're right, I've removed it!